xdp-dump/xdpdump.c

```diff
@@ -847,6 +1051,12 @@ int main(int argc, char **argv)
 			       &defaults_dumpopt) != 0)
 		return EXIT_FAILURE;
 
+	/* Do we need to dump version information? */
+	if (cfg_dumpopt.version) {
```

Comment:
For this I think we should just add a version output to params.c (to go along with --help and --verbose).

Reply:
It makes sense if we decide to use the same version for all apps and libxdp.

Reply:
Will add a --version in the param.c